filament/src/PostProcessManager.cpp

```diff
@@ -1027,17 +1028,17 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::colorGrading(FrameGraph& fg,
         }
     }
 
-    auto& ppColorGrading = fg.addPass<PostProcessColorGrading>("colorGrading",
+    auto& ppColorGrading = fg.addPass<PostProcessColorGrading>("colorGradingOptions",
```

Comment:
That's a strange name for a pass?

Reply:
Argh automatic refactoring that was a bit too eager. Will fix.